yearn/yearn.py

```diff
@@ -52,23 +53,43 @@ def __init__(self, load_strategies=True, load_harvests=False, watch_events_forev
             self.registries["v2"].load_harvests()
         logger.info('loaded yearn in %.3fs', time() - start)
 
+
+    def active_vaults_at(self, block=None):
+        active = set()
+        registries = self.registries.items()
+        for label, registry in registries:
+            if label == 'earn':
+                active = active.union({vault.vault for vault in registry.active_vaults_at_block(block=block)})
+            else:
+                active = active.union({vault.vault for vault in registry.active_vaults_at(block=block)})
+        try:
+            active.remove(contract("0xBa37B002AbaFDd8E89a1995dA52740bbC013D992")) # [yGov] Doesn't count for this context
```

Comment:
if you use `discard` instead of `remove`, you can drop the `try...except`

Reply:
eliminated this snippet while refactoring one of the above change requests